scripts/verify.sh

```diff
@@ -136,6 +160,64 @@ if [ $final_ret -eq 0 ]; then
         final_ret=7
     fi
 
+    # vw3 becomes an IBSS and then joins the "Elian" network (BSSID: 00:76:77:33:00:00).
```

Comment:
Remove your name in the message. Specify other names such as [Foobar](https://en.wikipedia.org/wiki/Foobar) if you have no idea.